src/store/index.ts

```diff
@@ -64,6 +65,7 @@ const reducers = combineReducers({
   [COOKIES_REDUCER_ID]: cookies,
   [ADDRESS_BOOK_REDUCER_ID]: addressBook,
   [CURRENT_SESSION_REDUCER_ID]: currentSession,
+  [TRANSACTIONS]: allTransactions,
 })
 
 export type AppReduxState = CombinedState<{
```

Comment:
could you please add reducer type?

Reply:
don't get you, could you add an example?

Reply:
The comment is literally on a LOC for redux state type defintitions